platform_config.yml

```diff
@@ -29,19 +29,25 @@ agents:
     source: $VOLTTRON_ROOT/services/core/VolttronCentral
     config: $VOLTTRON_ROOT/services/core/VolttronCentral/config
 
-  master.driver:
+  platform.driver:
     source: $VOLTTRON_ROOT/services/core/MasterDriverAgent
+    config_store:
+      fake.csv:
+        file: $VOLTTRON_ROOT/examples/configurations/drivers/fake.csv
+        type: --csv
+      devices/fake-campus/fake-building/fake-device:
+        file: $VOLTTRON_ROOT/examples/configurations/drivers/fake.config
 
   platform.agent:
```

Comment:
Since agents are installed sequentially,  I realized that the platform agent needs to be built after the voltton central agent is created because platform agent needs the `bacpypes` dependency, which it doesn't install on its own. However, volttron central agent does install that dependency.